packages/edit-widgets/src/components/header/undo-redo/undo.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { undo as undoIcon } from '@wordpress/icons';
+import { displayShortcut } from '@wordpress/keycodes';
+
+export default function UndoButton() {
```

Comment:
Is there a package these components could go in so we aren't duplicating code between edit post, edit site, and edit widgets?

Reply:
Maybe `block-editor` or alternatively `interface`?

Reply:
I wouldn't bother sharing these for the moment personally.